### PR TITLE
Update TimePickerModal.tsx

### PR DESCRIPTION
**Pull request by zkteco-home**
for customizing icon,for ex :
const  keyboard-outline= require('../assets/images/keyboard-outline.png'); clockIcon={()=><Image source={keyboard-outline}/>}

### CODE REVIEW

File: src/Time/TimeInput.tsx [hunk 1]
```diff
@@ -130,6 +130,7 @@ const styles = StyleSheet.create({
     textAlign: 'center',
     textAlignVertical: 'center',
     width: 96,
+    padding:0
```
Comment (SoyDiego) [inline src/Time/TimeInput.tsx:133]: Format code

**Review by SoyDiego — APPROVED** [review]

File: src/Time/TimePickerModal.tsx [hunk 2]
```diff
@@ -70,8 +70,8 @@ export function TimePickerModal({
   onDismiss: () => any
   onConfirm: (hoursAndMinutes: { hours: number; minutes: number }) => any
   animationType?: 'slide' | 'fade' | 'none'
-  keyboardIcon?: string
-  clockIcon?: string
+  keyboardIcon?: IconSource
```
Comment (SoyDiego) [inline src/Time/TimePickerModal.tsx:73]: You should edit also the documentation adding this change

File: src/Time/TimeInput.tsx (hunk 1 above)
Comment (SoyDiego) [inline src/Time/TimeInput.tsx:133]: I tried this code in `Android` and is doing anything (I don't know in `iOS`.
I think maybe is for your use case the problem

File: src/Time/TimePickerModal.tsx (hunk 2 above)
Comment (SoyDiego) [inline src/Time/TimePickerModal.tsx:73]: Can you add this change in the Docs page?
https://web-ridge.github.io/react-native-paper-dates/docs/time-picker

And also if you change this type, you

Comment (zkteco-home) [reply src/Time/TimePickerModal.tsx:73]: > Can you add this change in the Docs page? https://web-ridge.github.io/react-native-paper-dates/docs/time-picker
> 
> And also if you change this `string` type, you should change this too:
> 
> image
> 
> Thanks :)

no need to change this string type,you can read code again

Comment (zkteco-home) [reply src/Time/TimePickerModal.tsx:73]: image
here icon is IconButton props,
image

Comment (SoyDiego) [reply src/Time/TimePickerModal.tsx:73]: > > Can you add this change in the Docs page? https://web-ridge.github.io/react-native-paper-dates/docs/time-picker
> > And also if you change this `string` type, you should change this too:
> > image
> > Thanks :)
> 
> no need to change this string type,you can read code again

Are you sure? 

image-1

Comment (zkteco-home) [reply src/Time/TimePickerModal.tsx:73]: very sorry for that,
image

**Comment by RichardLindhout** [comment]
I'm not sure if it's good to import from react-native-paper/lib maybe we can add any instead of IconSource for now

**Pull request closed without merging**